GOVERNANCE.md

```diff
@@ -0,0 +1,253 @@
+As users continue to engage and become more involved with the Flux community, they may find themselves becoming contributors.
+
+### Contributors
+
+Flux welcomes all kinds of contributions, including code, issues, documentation, external tools, advocacy and community work.
+Contributors are accepted based on the merit of their contributions.
+Contributions help Maintainers ensure that the project continues to satisfy end user needs.
+Teams defined below may have different contribution processes and requirements for the various areas of Flux appropriate to their team.
+
+### Maintainers
+
+Maintainers are elected Contributors who serve on one or more Teams.
+For the election process, see Membership Changes below.
+
+Maintainers are expected to:
+
+- enable and promote Flux community Values
+- engage with end Users through appropriate communication channels appropriate to their teams
+- serve as a point of conflict resolution between Contributors to the Flux area appropriate to their teams
+- maintain open collaboration within and across teams
+- additional responsibilites are defined per team below
+
+## Teams
+
+The Flux project is governed by Maintainer teams, each with defined scope and resonsibilities.
+Teams are responsible for the overall organization, code development, community management, and security.
+
+The goal of governing by teams is to allow parts of the project to act independently as appropriate, enabling the project to grow horizontally while remaining well-maintained.
+
+Teams below are also mapped to GitHub teams within the `fluxcd` GitHub org.
+Their members are mapped to GitHub user accounts.
+
+### Org team
+
+@fluxcd/org <https://github.com/orgs/fluxcd/teams/org>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilities, as well as:
+- overseeing the project health and growth
+- maintaining the brand, mission, vision, values, and scope of the overall project
+- changes to licensing and intellectual property
+- administering access to all project assets
+- deleting source code repositories
+- handling code of conduct violations
+- managing financial decisions
+- defining the scope and responsibilities of all other teams
+- making decisions that affect multiple teams
+- resolving escalated team decisions when the teams responsible are blocked
+- changes to governance (this document)
+
+### Architecture team
+
+@fluxcd/arch <https://github.com/orgs/fluxcd/teams/arch>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilities, as well as:
+- administering and creating source code repositories
+- technical oversight on the overall project
+- making technical decisions that are applicable to multiple sub-projects
+- providing architectural guidance to Dev team members on an individual sub-project
+- representing sub-project interests in Architecture team decisions
+
+Every sub-project should have at least one Architecture team member as a Maintainer.
```

Comment:
I can see the intent of this, which is a sort of representative democracy; but what I'd expect to happen is a polarisation:

- for some subprojects the maintainers are more or less the architecture team, so a decision about the architecture is really a decision about those subprojects;
- some other subprojects get a passive maintainer, at odds with the the other maintainers, but without any particular stake in the subproject itself.

Reply:
This has now been removed